--- cvsfs/Makefile.org	Thu Nov 15 22:42:29 2001
+++ cvsfs/Makefile	Fri Nov 16 11:20:08 2001
@@ -25,7 +25,7 @@
 
 KERNEL_VERSION = $(shell uname -r)
 
-CFLAGS = -O3 -Wall -DMODULE -D__KERNEL__ -DLINUX -fomit-frame-pointer -I/lib/modules/$(KERNEL_VERSION)/build/include
+CFLAGS = -O3 -Wall -DMODULE -D__KERNEL__ -DLINUX -fomit-frame-pointer -I/usr/src/linux-$(KERNEL_VERSION)/include
 CXXFLAGS =
 LDFLAGS = -m elf_i386 -r
 
